packages/dashboard/src/components/dashboard/index.test.tsx

```diff
@@ -10,6 +10,7 @@ describe('Dashboard', () => {
     document.body.appendChild(container);
 
     const args: IotDashboardProps = {
+      query: undefined,
```

Comment:
Is this related in any way to the query we'll use to build timeseries data?

Reply:
Yep, same thing! [It's the query from `source-iotsitewise`.](https://github.com/awslabs/iot-app-kit/blob/main/docs/AWSIoTSiteWiseSource.md) You should be able to hook into it to get timeseries data too.

Reply:
cool. I'll plan on replacing the test query I've been using with this!